app/helpers/notifs.py

```diff
@@ -0,0 +1,14 @@
+"""sends a notification to discord"""
+import os
```

Comment:
Todo: Add a comment above the imports to specify the internal/external/file imports for consistency